templates/info.txt.j2

```diff
@@ -13,7 +13,8 @@ Congratulations, you have successfully installed home.{{ work_domain }} project.
   After full install manually trusts root certificate in the system!
 
      - Run this command in the project directory:
-       sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain docs/installation/ssl/tenantcloud-rootCA.crt
+       bash -lc "cd {{ project_path }} && \
```

Comment:
`project_path` or `$project_path` ?

Reply:
this variable is passed by the ansible, when generating a file, there will be a full path